homeassistant/components/ovo_energy/config_flow.py

```diff
@@ -9,10 +9,11 @@
 from homeassistant.config_entries import ConfigFlow
 from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
 
-from .const import CONF_ACCOUNT_ID, DOMAIN  # pylint: disable=unused-import
+from .const import DOMAIN  # pylint: disable=unused-import
 
 _LOGGER = logging.getLogger(__name__)
 
+REAUTH_SCHEMA = vol.Schema({vol.Required(CONF_PASSWORD): str})
```

Comment:
Just thinking... the username = email. So if you change your email address... ?

Reply:
Well looks like you can also use your 'my ovo id'

![image](https://user-images.githubusercontent.com/28114703/98444358-2854cb00-2109-11eb-8e1a-93b9efebd280.png)